src/one_dragon_qt/widgets/setting_card/common_download_card.py

```
@@ -104,6 +104,53 @@ def set_options_by_list(self, options: List[ConfigItem]) -> None:
         for opt_item in options:
             self.combo_box.addItem(opt_item.ui_text, userData=opt_item.value)
 
+    def _get_downloader_param(self, index: int | None = None) -> CommonDownloaderParam:
+        """
+        获取下载器参数
+        :param index: 选择的下标，如果为 None 则使用当前选中的下标
```

Ruff (0.14.1)

Docstring contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF002)